content/docs/rfcs/1/README.md

```diff
@@ -165,5 +166,6 @@ to specification metadata.
 | **shortname**    | short name           | string | 1/COSS              |
 | **name**         | full name            | string | Consensus-Oriented Specification System |
 | **status**       | status               | string | draft               |
+| **category**     | category             | string | application         |
 | **editor**       | editor name/email    | string | Oskar Thoren <oskar@status.im> |
-| **contributors** | contributors         | list   | Pieter Hintjens <ph@imatix.com>, André Rebentisch <andre@openstandards.de>, Alberto Barrionuevo <abarrio@opentia.es>, Chris Puttick <chris.puttick@thehumanjourney.net>, Yurii Rashkovskii <yrashk@gmail.com> |
+| **contributors** | contributors         | list   | - Dean Eigenmann <dean@status.im><br> - Andrea Maria Piana <andreap@status.im><br> - Oskar Thorén <oskar@status.im> |
```

Comment:
Why are you removing contributors who wrote all this and adding two people who had nothing to do with this spec?

Reply:
Happy to keep the names. What matters to me is the format. In all specs, we use itemized list to format contributors:
```
- Alice
- Bob
```

Where this table indicates that contributors should be on inline:

```
Alice, Bob
```

My aim is to align the COSS with the actual practice.

Reply:
Ah I see, uhm sure :D